spec/game_spec.rb

```diff
@@ -14,4 +14,9 @@
   expect(game.score).to eq 0
 end
 
+it 'can do' do
```

Comment:
As before would be good to show that this is testing that it can add up correctly for a game with no strikes or spares